### PR TITLE
Made changes to release as a new gem

### PR DESCRIPTION
**Pull request by manikandan0603**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,92 +8,5 @@ Can be used in place of Ruby's Logger
 
 In fact, it (currently) returns an instance of Logger.
 
-Forked from [loggiler](https://github.com/freeformz/logglier).
+Forked from (<https://github.com/zshannon/cloudwatchlogger>).
 
-Usage
```
Comment (mikevic) [inline README.md:13]: @manikandan0603 - why did you delete the readme bro? 😓

Comment (manikandan0603) [reply README.md:13]: Since we are gonna use notion doc for implementation, I removed the old content.  What do you suggest we do here?

Comment (mikevic) [reply README.md:13]: I suggest add the stuff from Notion here? It should be possible to use this repo on its own, if anyone else also wants to use it, since the gem is public

Comment (manikandan0603) [reply README.md:13]: @mikevic - Updated the Readme.

**Review by mikevic — APPROVED** [review]
